examples/abitest/config/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+readonly MODULES_DIR="${DIR}/../../target/wasm32-unknown-unknown/release"
+readonly OUT_DIR="${DIR}/../bin"
```

Comment:
Are we settling on `bin` as the output directory? Why not... `out`? No need to change it anyways, we'll see later on.

cc @ipetr0v

Reply:
I like the name bin better, as it is a place where we dump binary artefacts. Out seems... strange.

Reply:
Is there an out directory for `oak_loader`?

Reply:
No, this is for the examples. I don't think the `oak_loader` should be there

Reply:
I think we should have it as `oak/server/bin/oak_loader` at some point.

Reply:
+1 to @tiziano88  suggestion